src/main/resources/application-localstack.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  profiles:
+    include: dev
+
+sns:
```

Comment:
why the `sns` configuration here?

Reply:
It shouldn't be. Left over from coping the pattern from offender-case-notes.